### ISSUE TITLE
Net64 and Project64 never connect

### ISSUE THREAD

**Issue by gfrhj** [Tarnadas/net64plus#111]
Nothing happens on the emulator at all. Just regular SM64 with nothing else added to it

**Comment by fancythedeveloper** [comment]
First, run both as Administrator.
Second, set Memory Size for SM64 as 16MB.
Third, open the game and run until you get to the file select screen.
Fourth, open Net64+ and connect the client to the emulator.
Assuming nothing is missed, it should work.

**Comment by gfrhj** [comment]
No, What I've had to do is re extract the client every time I run Net64
